src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -28,12 +28,12 @@ feedback:
 ## Introduction
 
 {% Aside 'note' %}
-  📚 I have done a lot of background research on the history and theory of dark mode,
-  if you are only interested in working with dark mode, feel free to
-  [skip the introduction](#activating-dark-mode-in-the-operating-system).
+📚 I have done a lot of background research on the history and theory of dark mode,
```

Comment:
```suggestion
I have done a lot of background research on the history and theory of dark mode,
```

Reply:
Asides already have a symbol in them. This is also agains the best practice of omitting things that don't contribute meaning to a sentence.